demo/api/src/products/entities/product-category.entity.ts

```diff
@@ -20,7 +20,7 @@ export class ProductCategory extends BaseEntity<ProductCategory, "id"> {
     @Field()
     title: string;
 
-    @Property()
+    @Property({ unique: true })
     @Field()
     slug: string;
```

Comment:
Note: This only works if the products aren't scoped (which is the case here, so it's fine)